README.adoc

```diff
@@ -1931,6 +1931,19 @@ end
 
 Prefer the use of exceptions from the standard library over introducing new exception classes.
 
+=== Do not `raise` `Exception` [[raise-exception]]
+
+Use `StandardError` over `Exception`.
```

Comment:
Perhaps this section should include a sentence about _why_ one should not `raise Exception`.

Reply:
Could you help with wording?